tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php

```diff
@@ -307,6 +307,10 @@ public function provideNotFixCases(): \Generator
         yield 'not in class' => [
             '<?php self::assertTrue(is_null($a));',
         ];
+
+        yield 'is_resource' => [
```

Comment:
this PR doesn't provide any reasoning for the change, this test case description also.
please add it here

Reply:
Added.